### PR TITLE
add github of contributors

### PR DESCRIPTION
**Pull request by vasudevgupta7**
This PR will add contributors GitHub id at the end of every dataset cards.

### CODE REVIEW

**Comment by vasudevgupta7** [comment]
@lhoestq Can you confirm if this format is fine? I will update cards based on your feedback.

**Comment by julien-c** [comment]
On HuggingFace side we also have a mapping of hf user => github user (GitHub info used to be required when signing up until not long ago – cc @gary149 @beurkinger) so we can also add a link to HF profile

File: datasets/afrikaans_ner_corpus/README.md [hunk 1]
```diff
@@ -166,4 +166,6 @@ The data is under the [Creative Commons Attribution 2.5 South Africa License](ht
   year      = {2016},
   url       = {https://repo.sadilar.org/handle/20.500.12185/299},
 }
-```
\ No newline at end of file
+```### Contributions
```
Comment (lhoestq) [inline datasets/afrikaans_ner_corpus/README.md:169]:
````suggestion
```

### Contributions
````

File: datasets/bsd_ja_en/README.md [hunk 2]
```diff
@@ -182,4 +182,6 @@ This dataset was released under the Creative Commons Attribution-NonCommercial-S
     doi = "10.18653/v1/D19-5204",
     pages = "54--61"
 }
-```
\ No newline at end of file
+```### Contributions
```
Comment (lhoestq) [inline datasets/bsd_ja_en/README.md:185]:
````suggestion
```

### Contributions
````

**Review by lhoestq — COMMENTED** [review]
The format looks good ! :) 

Maybe you can add an empty line before each `Contributions` section.
We should also edit the templates to include this new section: the [README](https://github.com/huggingface/datasets/blob/master/templates/README.md) template and the [README guide](https://github.com/huggingface/datasets/blob/master/templates/README_guide.md) template.

@julien-c how can we get the mapping between hf users and github users ?

**Comment by vasudevgupta7** [comment]
All the dataset cards have been updated with GitHub ids :)

**Review by lhoestq — APPROVED** [review]
Thank you !!
LGTM

**Pull request merged**